docs/operating-eck/upgrading-eck.asciidoc

```diff
@@ -11,6 +11,20 @@ endif::[]
 [id="{p}-ga-upgrade"]
 == Upgrade to ECK {eck_version}
 
+[IMPORTANT]
+================================
+This is the upgrade guide for ECK. For Elastic Stack upgrade, please refer to this <<{p}-upgrading-stack,guidance>> for more details.
+================================
+
+[CAUTION]
+================================
+Upgrading the ECK operator results in a one-time update to existing managed resources in the cluster. This triggers a rolling restart of pods, on which Elastic Stack components are running, by Kubernetes to apply those changes. Therefore, if you have a large Elasticsearch cluster or multiple Elastic Stack deployments, this rolling restart will cause performance degradation, which might be disruptive or inconvenient.
```

Comment:
I don't think a performance degradation is a necessary consequence of every rolling restart. I think it depends on the use case.  
```suggestion
Upgrading the ECK operator results in a one-time update to existing managed resources in the cluster. This triggers a rolling restart of pods, on which Elastic Stack components are running, by Kubernetes to apply those changes. Therefore, if you have a large Elasticsearch cluster or multiple Elastic Stack deployments, this rolling restart could cause a performance degradation, which might be disruptive or inconvenient.
```

Reply:
Thanks! Indeed it depends on the use case!